tracer/src/lib.rs

```diff
@@ -313,6 +313,10 @@ pub fn decode(elf: &[u8]) -> (Vec<RV32IMInstruction>, Vec<(u64, u8)>, u64, Xlen)
                 if (first_halfword & 0b11) != 0b11 {
                     // Compressed 16-bit instruction
                     let compressed_inst = first_halfword;
+                    if compressed_inst == 0x0000 {
+                        offset += 2;
+                        continue;
+                    }
```

Comment:
Does such thing ever happen in practice?
I'm pretty sure if we just skip it here our verification will fail. This is basically a compresed noop instruction, and we can't have noops in the bytecode/trace at all (except for padding).